boards/arduino-atmega-common/include/arduino_pinmap.h

```diff
@@ -156,6 +156,26 @@ extern "C" {
 #define ARDUINO_PIN_A15         ARDUINO_PIN_69
 #endif
 
+/* arduino ADC notation compatibility */
+#define A0                     0
+#define A1                     1
+#define A2                     2
+#define A3                     3
+#define A4                     4
+#define A5                     5
+#ifdef CPU_ATMEGA2560
+#define A6                     0
```

Comment:
are the defines for `A6` to `A9` correct, 'cause they are the same as `A0` to `A3` and would reuse same ADC channels?